### ISSUE TITLE
28493092: UIBarButtonItem accessibilityLabel should be configurable in IB

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#15966]
#### Description

There's currently no way to specify the accessibilityLabel of UIBarButtonItem in IB.

I tried setting `accessibilityLabel` as a User Defined Runtime Attribute with type `Localized String`. Xcode correctly adds the localized string to the xxx.strings file but iOS fails to localize the accessibilityLabel according to the provided translation.

Note:
I created an ugly workaround by swizzling -[UIBarButtonItem  awakeFromNib] but this is such a basic and essential feature this should really not be needed!
- (void)myAwakeFromNib {
  [self myAwakeFromNib];
  
  if (self.accessibilityLabel.length) {
      self.accessibilityLabel = [[NSBundle mainBundle] localizedStringForKey:self.accessibilityLabel value:self.accessibilityLabel table:nil];
  }
  }
## 

Product Version: 10.1 beta 1
Created: 2016-09-27T11:15:07.421630
Originated: 2016-09-27T13:14:00
Open Radar Link: http://www.openradar.me/28493092
